### ISSUE TITLE
Error on .blade.php does not exist 

### ISSUE THREAD

**Issue by rognales** [imanghafoori1/laravel-microscope#48]
My stack as follows:-

PHP 7.4
Laravel Framework 6.18.10
laravel-microscope 1.0

Errors reported by **Laravel Microscope**:-

````php
.blade.php does not exist 
public function view(): View at app\Exports\PriceExport.php:27
````

Snippets of ```PriceExport.php```

````php
public function view(): View
{
    return view('export.main');
}
````

The blade file````export.main```` do exists

**Comment by imanghafoori1** [comment]
Hi
which patch release you have ?
I think this issue is already fixed in the `1.0.8` version or above.

**Comment by rognales** [comment]
My version as below.

````
λ composer show imanghafoori/laravel-microscope
name     : imanghafoori/laravel-microscope
descrip. : Auto test your laravel application
keywords : laravel-microscope, laravel-testing
versions : * v1.0.0
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
homepage : https://github.com/imanghafoori1/laravel-microscope
source   : [git] https://github.com/imanghafoori1/laravel-microscope.git 82f6c1389c8bdf68294875f4ccf3f520d04c992c
dist     : [zip] https://api.github.com/repos/imanghafoori1/laravel-microscope/zipball/82f6c1389c8bdf68294875f4ccf3f520d04c992c 82f6c1389c8bdf68294875f4ccf3f520d04c992c
path     : C:\laragon\www\sms\vendor\imanghafoori\laravel-microscope
names    : imanghafoori/laravel-microscope

autoload
psr-4
Imanghafoori\LaravelMicroscope\ => src

requires
laravel/framework 5.*|6.*|7.*

requires (dev)
orchestra/testbench ^4.0
phpunit/phpunit ^8.0
````

**Comment by imanghafoori1** [comment]
it seems you have v1.0.0
so please upgrade to v1.0.8

**Comment by rognales** [comment]
Nevermind. Closed due to outdated version. Fixed on 1.0.8
